src/classes/artifact.ts

```diff
@@ -36,14 +37,7 @@ export class Artifact {
       ? `${type}-${this.version}-${LIB_VERSION}`
       : `${type}-${this.version}`
     this.archive = `${this.name}.zip`
+    this.folder = `${this.name}`
     this.scriptName = osCheck(`${baseScriptName}.bat`, `./${baseScriptName}`)
   }
-
```

Comment:
FYI I filed https://github.com/apache/daffodil-vscode/issues/711 which is related to these lines.

Reply:
@arosien Just pushed an update to also delete the `Backend` class and linked to close issue #711